sdk/deviceupdate/iot-device-update/samples/v1/javascript/updates.js

```diff
@@ -10,8 +16,8 @@ const instanceId = process.env["INSTANCE_ID"] || "<INSTANCE_ID>";
 /**
  * Get a list of all update providers that have been imported to Device Update for IoT Hub
  */
-export async function main() {
-  console.log("JS");
+async function main() {
+  console.log("TS");
```

Comment:
why?

Reply:
That's a good question. This is completely auto generated, just ran `dev-tool samples publish` again. I'm going to update this by hand

@willmtemple - fyi

Reply:
NVM this is hard coded in the sample itself. I'll remove that log line